RELEASE-NOTES.md

```diff
@@ -1,119 +1,135 @@
+* The following log rotation items have been added to cuttlefish:
+    * log.console.size
+    * log.console.rotation
+    * log.console.rotation.keep
+    * log.error.rotation
+    * log.error.rotation.keep
+    * log.error.size
+
+[[PR 1164](https://github.com/basho/riak_cs/pull/1164) and [PR 97](https://github.com/basho/stanchion/pull/97)]
+
+* `riak_cs_wm_common` now has a default callback of `multiple_choices`, which prevents `code_server` from becoming a bottleneck. [[PR 1181](https://github.com/basho/riak_cs/pull/1181)]
+* An option has been added to replace the `N=all user GET` option with `N=quorum` just before authentication. This option improves latency, especially in the presence of slow (or actually-failing) nodes blocking the whole request flow because of N=all. When enabled, a user's owned-bucket list is never pruned after a bucket is deleted, instead it is just marked as deleted. [[PR 1191](https://github.com/basho/riak_cs/pull/1191)]
+* An info log has been added when starting a storage calculation batch. [[PR 1238](https://github.com/basho/riak_cs/pull/1238)]
+* `GET Bucket` requests now have clearer responses. A 501 stub for Bucket lifecycle and a  simple stub for Bucket requestPayment have been added. [[PR 1223](https://github.com/basho/riak_cs/pull/1223)]
+* Several user-friendly features have been added to [`riak-cs-debug`](http://docs.basho.com/riakcs/latest/cookbooks/command-line-tools/): fine-grained information gathering options, user-defined filtering for configuration files, and verbose output for failed commands. [[PR 1236](https://github.com/basho/riak_cs/pull/1236)]
+
+###Enterprise
+* MDC has `proxy_get`, which make block objects propagate to site clusters when they are requested. Now, multibag configuration with MDC supports `proxy_get`. [[PR 1171](https://github.com/basho/riak_cs/pull/1171) and [PR 25](https://github.com/basho/riak_cs_multibag/pull/25)]
+* Configuration options to replicate blocks in realtime replication (`replicate_cs_blocks_realtime`) and to replicate block tombstones in realtime replication (`replicate_cs_block_tombstone`) are now available. `replicate_cs_blocks_realtime` relieves the load in fullsync replication (there will be a longer realtime queue in realtime replication and risk of dropping manifests or other important keys), and it is set to `false` by default. `replicate_cs_block_tombstone` causes source block deletes caused by garbage collection to be replicated to the sink cluster. This option is set to `true` by default. Both configuration options are set in the advanced.config of the Riak node, rather than in the Riak CS configuration. [[PR 678](https://github.com/basho/riak_repl/pull/678)]
```

Comment:
If we keep this line here, we need the link, and a bit update for the text because `replicate_cs_block_tombstone` has changed.

This code is included in Riak, not in CS. But this is **only for** CS. It will be shipped in Riak 2.1.2, but not included in Riak 2.1.1. I don't have good recommendation about where to mention it.